src/main/java/br/com/codenation/squad1/util/exceptionhandler/StandardError.java

```diff
@@ -0,0 +1,53 @@
+package br.com.codenation.squad1.util.exceptionhandler;
+
+import java.io.Serializable;
+
+public class StandardError implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private Long timestamp;
+    private Integer status;
+    private String message;
+    private String path;
+
+    public StandardError(Integer status, String message, String path) {
+        super();
+        this.timestamp = System.currentTimeMillis();
```

Comment:
preferencia utilizar bibliotecas do java8

Reply:
alterado para 
this.timestamp = Instant.now().toEpochMilli();